sphinx_gallery/clean.py

```diff
@@ -0,0 +1,29 @@
+from argparse import ArgumentParser
+import os.path
+import shutil
+import subprocess
+import sys
+
+
+def main():
+    parser = ArgumentParser(
+        description=
+        "Clean intermediate files generated by sphinx-gallery, and call "
+        "'python -msphinx -M clean' with the same arguments.")
+    parser.add_argument("sourcedir")
+    parser.add_argument("builddir")
+    args, _ = parser.parse_known_args()
+    d = {}
+    with open(os.path.join(args.sourcedir, "conf.py")) as file:
+        exec(file.read(), d)
```

Comment:
`file` is a builtin in python 2 please use: `sg_conf_file`, `sg_conf_py` or other reasonable variable name.

Reply:
fixed, also needed to backport exec for Py2...

Reply:
where is this fixed?